tests/core/evm/vm/test_message.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import evm.vm.message as m
+
+message_object = m.Message(
```

Comment:
just added a commit 'added create message factory as commented' to address this comment.  Hitting each one individually.